modules/api/src/main/java/org/apache/fluo/api/client/FluoAdmin.java

```diff
@@ -134,6 +132,8 @@ void initialize(InitializationOptions opts) throws AlreadyInitializedException,
    */
   void updateSharedConfig();
 
+  FluoConfiguration getSharedConfig();
```

Comment:
Could have two methods here.  One that returns exactly what config was passed to the factory and another that passes what config is stored in zookeeper.  We could create another issue to support merging one simple config into another.  This would be a good starter issue.